system/lib/wasmfs/js_api.cpp

```diff
@@ -270,6 +270,10 @@ int _wasmfs_lstat(char* path, struct stat* statBuf) {
   return __syscall_lstat64((intptr_t)path, (intptr_t)statBuf);
 }
 
+int _wasmfs_mkdev(char* path, mode_t mode, backend_t backend) {
+  return wasmfs_create_file(path, mode, backend);
```

Comment:
Ah, nice this is so simple :smile: 

Actually, perhaps we can just use the existing `wasmfs_create_file`, and not add this?